.pre-commit-config.yaml

```diff
@@ -9,6 +9,7 @@ repos:
     rev: 21.6b0
     hooks:
     -   id: black
+        exclude: ^examples
```

Comment:
Otherwise, pre-commit is blackifying the examples as well. We might want to avoid this for the moment and minimize the diff.